chat/be/server.js

```diff
@@ -208,12 +208,24 @@ async function run() {
 
     io.emit('message', newMessage);
 
+    const userRoomConfigs = await UserRoomConfig.find(
```

Comment:
Instead of a new emit, can the FE just use the existing message emit to increment the number of unread messages?